res/enkf/local_obsdata.py

```diff
@@ -1,138 +1,3 @@
-        else:
-            raise ValueError(
-                'Unable to construct LocalObsdata with name "%s" from given obs.' % name
-            )
-
-    def initObservations(self, obs):
-        self.obs = obs
-
-    def __len__(self):
-        """@rtype: int"""
-        return self._size()
-
-    def __getitem__(self, key):
-        """@rtype: LocalObsdataNode"""
-        if isinstance(key, int):
-            if key < 0:
-                key += len(self)
-            if 0 <= key < len(self):
-                return _lib.local.local_obsdata.iget_node(self, key)
-            else:
-                raise IndexError("Invalid index, valid range is [0, %d)" % len(self))
-        else:
-            if key in self:
-                return _lib.local.local_obsdata.get_node(self, key)
-            else:
-                raise KeyError('Unknown key "%s".' % key)
-
-    def __iter__(self):
-        cur = 0
-        while cur < len(self):
-            yield self[cur]
-            cur += 1
-
-    def __contains__(self, item):
-        """@rtype: bool"""
-        if isinstance(item, str):
-            return self._has_node(item)
-        elif isinstance(item, LocalObsdataNode):
-            return self._has_node(item.getKey())
-
-        return False
-
-    def __delitem__(self, key):
-        assert isinstance(key, str)
-        if key in self:
-            self._del_node(key)
-        else:
-            raise KeyError('Unknown key "%s".' % key)
-
-    def addNode(self, key):
-        """@rtype: LocalObsdataNode"""
-        assert isinstance(key, str)
-        if key in self.obs:
-            if key not in self:
-                node = LocalObsdataNode(key)
-                return _lib.local.local_obsdata.add_node(self, node)
-            else:
-                raise KeyError("Tried to add existing observation key:%s " % key)
-        else:
-            raise KeyError(
-                "The observation node: %s is not recognized observation key" % key
-            )
-
-    def addObsVector(self, obs_vector):
-        self.addNode(obs_vector.getObservationKey())
-
-    def name(self):
-        return self._name()
-
-    def getName(self):
-        """@rtype: str"""
-        return self.name()
-
-    def getActiveList(self, key):
-        """@rtype: ActiveList"""
-        if key in self:
-            return _lib.local.local_obsdata.get_active_list(self, key)
-        else:
-            raise KeyError('Local key "%s" not recognized.' % key)
-
-    def copy_active_list(self, key):
-        """@rtype: ActiveList"""
-        if key in self:
-            return _lib.local.local_obsdata.copy_active_list(self, key)
-        else:
-            raise KeyError('Local key "%s" not recognized.' % key)
-
-    def free(self):
-        self._free()
-
-    def __repr__(self):
-        return "LocalObsdata(len = %d, name = %s) at 0x%x" % (
-            len(self),
-            self.name(),
-            self._address(),
-        )
+__all__ = ["LocalObsdata"]
```

Comment:
Think this has a few integration test, but very few unit tests, would be good to add some to make sure the behavior does not change.

Reply:
OK: this is tested here: https://github.com/equinor/ert/pull/3046